lib/routes/deadbydaylight/index.ts

```diff
@@ -0,0 +1,69 @@
+import { Route } from '@/types';
+import cache from '@/utils/cache';
+import ofetch from '@/utils/ofetch';
+import { parseDate } from '@/utils/parse-date';
+import MarkdownIt from 'markdown-it';
+const md = MarkdownIt({
+    html: true,
+    linkify: true,
+});
+
+const baseUrl = 'https://deadbydaylight.com';
+
+export const route: Route = {
+    path: '/blog',
+    categories: ['game'],
+    example: '/deadbydaylight/blog',
+    parameters: {},
+    features: {
+        requireConfig: false,
+        requirePuppeteer: false,
+        antiCrawler: false,
+        supportBT: false,
+        supportPodcast: false,
+        supportScihub: false,
+    },
+    radar: [
+        {
+            source: ['deadbydaylight.com/news'],
+            target: '/deadbydaylight/news',
```

Comment:
```suggestion
            target: '/news',
```
Only `example` needs namespace to be prepended.